autodispose-rxlifecycle/src/main/java/com/ubercab/autodispose/rxlifecycle/package-info.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2017. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Interop for RxLifecycle.
```

Comment:
Please write a more detailed doc, this is the doc for the entire package

Reply:
Added more details  and license - https://github.com/uber/AutoDispose/pull/118/commits/e463be672369a0f758516ad0cc61614f1cc67035